clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/description/MigrateLoadBalancerDescription.groovy

```diff
@@ -30,8 +30,6 @@ class MigrateLoadBalancerDescription {
   boolean dryRun
 
   @JsonIgnore
-  NetflixAmazonCredentials getCredentials() {
-    return target.getCredentials();
-  }
+  Set<AccountCredentials> credentialsList = [];
```

Comment:
Not sure I'm a big fan of `credentialsList`.

I would how bad it would be to keep it as `Set<NetflixAmazonCredentials> credentials = []` and do a check in `DefaultAllowedAccountsValidator` for whether it's a `Collection` or not.